applications/OptimizationApplication/python_scripts/utilities/optimization_problem.py

```diff
@@ -28,7 +30,9 @@ def __init__(self, echo_level: int = 0) -> None:
         self.__components: 'dict[Any, dict[str, Any]]' = {
             ResponseFunction: {},
             ExecutionPolicy: {},
-            Control: {}
+            Control: {},
+            MasterControl:{},
+            ResponseRoutine:{}
```

Comment:
The hints in line 48, 55, 68 could be extended to include MasterControl and ResponseRoutine.

Reply:
Thnx!